Translation of email and tweets to Portuguese (#15)
outreachy/creative-works-and-scripts: creative-works/sample-email-PORTUGUESE.txt

Olá,

> Inscrições estão abertas agora para o Outreachy, um programa de estágio que apoia diversidade em software livre e de código aberto (open source). Outreachy oferece estágios remotos, com duração de três meses, com bolsa de $5,500 e um auxílio de $500 para viagem. Outreachy expressamente convida a se candidatarem pessoas que são mulheres (tanto cis quanto trans), homens trans, e pessoas genderqueer. Nós também convidamos expressamente pessoas residentes e nativas dos E.U.A. de qualquer gênero, que sejam Negras/Afro-americanas, Hispânicas/latinas, Nativo americanas/Índio americanas, nativo alasquianas, nativo havainas ou ilhéus do Pacífico. Qualquer pessoa que sofram de preconceito ou discriminação na área de tecnologia em seus países também é convidada a aplicar.

Reviewer: While that is a perfect translation of what was originally written, it is sounding a bit weird in Portuguese. My suggestion would be:

```suggestion
As inscrições para o Outreachy, um programa de estágio que apoia a diversidade em projetos de software livre e código aberto, agora estão abertas. Outreachy oferece estágios remotos com duração de três meses, uma bolsa de U$5.500 e um auxílio-financeiro de U$500 para viagens, e expressamente convida a se candidatarem mulheres (tanto cis quanto trans), homens trans, e pessoas genderqueer. Nós também convidamos expressamente pessoas residentes e nativas dos E.U.A. de qualquer gênero, que sejam Negras/Afro-americanas, Hispânicas/latinas, Nativo americanas/Índio americanas, nativo alasquianas, nativo havainas ou ilhéus do Pacífico. Qualquer pessoa que sofram de preconceito ou discriminação na área de tecnologia em seus países também é convidada a se candidatar.
```
Changes:
- Text is now sounding more natural in Portuguese.
- Added U$ to make it clear all stipend is in US dollars.
- Localized 5,500 as in Brazil we use dots, not commas.
- In Portuguese, the repetition of the word "Outreachy" makes the text a bit exhausting to read. Added an "(...) and expressively (...)" to the beginning of the second phrase using it to make it a bit better.
- Deleted the first "pessoas que são (...)" as it seems to be unnecessary.
- "Se aplicar" also sounds a bit unnatural. Unfortunately, using "se candidatar" (repeating the same verb as before) seems to be is the only option to me.

> Estagiários do Outreachy trabalham remotamente com mentores experientes em projetos de software livre e de código aberto. Os estágios são normalmente focados em atividades de programação, mas alguns projetos oferecem estágios em experiência do usuário, design gráfico, documentação, desenvolvimento web, marketing, tradução, entre outros.

Reviewer: This part is almost perfectly translated. I would only change two things:

```suggestion
Estagiários do Outreachy trabalham remotamente com mentores experientes em projetos de software livre e de código aberto. Os estágios são normalmente focados em atividades de programação, mas alguns projetos oferecem estágios relacionados a experiência de usuário, design gráfico, documentação, desenvolvimento web, marketing, tradução, entre outros.
```
- Add "related to" ("relacionados a" in Portuguese).
- Not use "do usuário" but "de usuário".

> Os estágios do Outreachy acontecem de 4 de Dezembro de 2018 até dia 4 de Março de 2019. Mais detalhes e os critérios de elegibilidade podem ser acessados aqui:

Reviewer: This is perfect. I would only delete "até dia", use "a" instead and not capitalize month days to conform to Portuguese grammatical rules:

```suggestion
Os estágios do Outreachy acontecem de 4 de dezembro de 2018 a 4 de março de 2019. Mais detalhes e os critérios de elegibilidade podem ser acessados aqui:
```

https://www.outreachy.org/apply/eligibility/

> Aplicar para o Outreachy é um pouco diferente do processo de outros programas de estágio. Durante o período de inscrição (de 19 de Setembro até 30 de Outubro), é esperado da pessoa candidata a entrar em contato com um ou mais mentores de projeto, e fazer uma contribuição para o projeto. Organizadores do Outreachy notaram que as pessoas canditadas mais preparadas são aquelas que entram em contato com o seu mentor antecipadamente, fazem várias perguntas, e enviam contribuições continuamente durante o período de inscrição.

Reviewer: Again the problem with the verb "apply".

```suggestion
Candidatar-se ao Outreachy é um pouco diferente do processo de outros programas de estágio. Durante o período de inscrição (de 19 de setembro a 30 de outubro), é esperado da pessoa candidata a entrar em contato com um ou mais mentores de projeto, e fazer uma contribuição. A organização do Outreachy notou que as pessoas canditadas mais fortes são aquelas que entram em contato com seus mentores antecipadamente, fazem várias perguntas, e enviam contribuições continuamente durante o período de inscrição.
```
Changes:
- "Aplicar" to "Candidatar-se".
- To make "organizers" gender neutral, I used "organização" (organization) instead so it wouldn't be exhausting to read "pessoas" this, "pessoas" that.
- "Strongest" became "prepared" in the translation but I think "fortes" is a great equivalent in Portuguese also.
- "o seu mentor" became "seus mentores" to keep it gender neutral too.

> Por favor, entre em contato caso tenham perguntas sobre o programa. As pessoas organizadoras do Outreachy (Karen Sandler, Sage Sharp, Marina Zhurakhinskaya, Cindy Pallares e Tony Sebro) podem ser contatadas a partir do formulário de contato no https://outreachy.org/contact

Reviewer: :+1: but

```suggestion
Por favor, entre em contato caso tenham dúvidas sobre o programa. As pessoas organizadoras do Outreachy (Karen Sandler, Sage Sharp, Marina Zhurakhinskaya, Cindy Pallares e Tony Sebro) podem ser contatadas a partir do formulário de contato no https://outreachy.org/contact
```
- "dúvidas" is a bit better than "perguntas" in this case.

> Esperamos que você nos ajude a compartilhar os estágios do Outreachy!

Reviewer: The original says "spread the word". The best equivalent I can find in Portuguese in this case is "divulgar". A more literal translation ("espalhar a palavra") has religious connotations.

```suggestion
Esperamos que você nos ajude a divulgar os estágios do Outreachy!
```
